<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlgnPlus">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlgnSubtraction">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlgnProduct">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlgnDivision">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum9">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDecimal">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNum0">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSlgnEqual">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClean">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackSpace">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/new/prefix1/img/arrow_back_outline_48px_1158503_easyicon.net.ico</normalon>
    </iconset>
   </property>
  </widget>
  <zorder>pushButtonNum3</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButtonSlgnPlus</zorder>
  <zorder>pushButtonSlgnSubtraction</zorder>
  <zorder>pushButtonSlgnProduct</zorder>
  <zorder>pushButtonSlgnDivision</zorder>
  <zorder>pushButtonNum7</zorder>
  <zorder>pushButtonNum4</zorder>
  <zorder>pushButtonNum1</zorder>
  <zorder>pushButtonNum8</zorder>
  <zorder>pushButtonNum2</zorder>
  <zorder>pushButtonNum5</zorder>
  <zorder>pushButtonNum9</zorder>
  <zorder>pushButtonNum6</zorder>
  <zorder>pushButtonDecimal</zorder>
  <zorder>pushButtonNum0</zorder>
  <zorder>pushButtonSlgnEqual</zorder>
  <zorder>pushButtonClean</zorder>
  <zorder>pushButtonBackSpace</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
